odoo/service/server.py

```diff
@@ -160,6 +160,12 @@ def end_headers(self, *a, **kw):
             self.rfile = BytesIO()
             self.wfile = BytesIO()
 
+    def log_error(self, format, *args):
+        if format == "Request timed out: %r" and config['test_enable']:
```

Comment:
```suggestion
        if format and format[0:18] == "Request timed out:" and config['test_enable']:
```
![Selection_4396](https://github.com/user-attachments/assets/d8009d69-4f37-44a6-bf41-4ee3624ac3af)

Reply:
```
>>> x = "a %r"
>>> x == "a %r"
True
```
https://github.com/python/cpython/blob/2c8f329dc634290fb88636f85c05e473bc0104d5/Lib/http/server.py#L434